applications/fishing-map/src/features/analysis/AnalysisGraph.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react'
+import {
+  ResponsiveContainer,
+  CartesianGrid,
+  XAxis,
+  YAxis,
+  Tooltip,
+  Line,
+  ComposedChart,
+  Area,
+} from 'recharts'
+import { format } from 'd3-format'
+import { DateTime } from 'luxon'
+import { formatI18nNumber } from 'features/i18n/i18nNumber'
+import i18n from 'features/i18n/i18n'
+import styles from './AnalysisGraph.module.css'
+
+export interface GraphData {
+  date: string
+  min: number
+  max: number
+}
+
+interface AnalysisGraphProps {
+  timeseries: GraphData[]
+  graphColor?: string
+  graphUnit?: string
+}
+
+const tickFormatter = (tick: number) => {
+  const formatter = tick < 1 ? '~r' : '~s'
+  return format(formatter)(tick)
+}
+
+const formatDates = (tick: string) => {
+  const tickDate = DateTime.fromISO(tick)
```

Comment:
This will display the date in the users's timezone, shouldn't we use always UTC? if so then you should use `DateTime.fromISO(tick).toUTC()`. In my case I see `29 Dec 19` instead of `30 Dec 19` that is the start date of the time range I selected

Reply:
✔️